applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py

```diff
@@ -2,6 +2,23 @@
 
 import KratosMultiphysics
 
+def CheckForParallelism(settings):
+    if not settings["problem_data"].Has("parallel_type"):
+        raise Exception("You must provide a \"parallel_type\" in your settings!")
```

Comment:
>  You must provide a "parallel_type" in your settings!
to 
> ERROR : "parallel_type" not specified in settings.
??
similarly for the other Exceptions in the following lines of code ??

Reply:
Yes, I check for it since it is needed for the solver selection